doc_swift_ui_gallery/swift_ui_gallery/swift_ui_gallery/layouts/VStackExamples.swift

```diff
@@ -2,66 +2,201 @@ import SwiftUI
 
 struct VStackTopAlignedPage : View {
     var body: some View {
-        VStack() {
-            Text("First")
-            Text("Second")
-            Text("Third")
-            
-            Spacer()
-        }
+        NavigationStack {
+            VStack() {
+                Text("First")
+                Text("Second")
+                Text("Third")
+                Spacer()
```

Comment:
Same comment here about the user of `Spacer`s

Reply:
As per the Apple docs HStack uses VerticalAlignment and VStack uses HorizontalAlignment.
Ref: https://developer.apple.com/documentation/swiftui/aligning-views-within-a-stack#Customize-stack-alignment

So if we need to use HStack with HorizontalAlignment, one way is to wrap the VStack items inside HStack and then use it. It is also shown in the docs here: https://developer.apple.com/documentation/swiftui/horizontalalignment

Here in the above example individual items of HStack are VStack elements which have been given, (.leading, .center, .trailing) alignment. Other way is to use the Spacer()